tests/test_ollama.py

```diff
@@ -147,3 +158,37 @@ def test_model_embed(
 def test_registered_models_when_ollama_is_down(mock_ollama_list):
     mock_ollama_list.side_effect = ConnectError("[Errno 111] Connection refused")
     assert not any(isinstance(m.model, Ollama) for m in get_models_with_aliases())
+
+
+@pytest.mark.skipif(IN_GITHUB_ACTIONS, reason="Test doesn't work in Github Actions.")
+@pytest.mark.asyncio
+async def test_actual_run():
+    """Tests actual run. Needs llama3.2"""
+    model = get_async_model("llama3.2:latest")
+    response = model.prompt("a short poem about tea")
+    response_text = await response.text()
+    assert len(response_text) > 0
+
+
+@pytest.mark.asyncio
+async def test_actual_run(mock_ollama):
```

Comment:
This redefines the previous test case unconditionally; should this be given a different name?